update references to use new CLI (#18878)
expo/expo: docs/pages/bare/using-expo-client.md

It's not currently possible to include your own native code in Expo Go, so it might surprise you to learn that it is still possible to run your bare project in the Expo Go app.

> Inside a freshly initialized bare project, run `npx expo start` and you can now run it in the client. Read on to learn more about the limitations, why you might want to still use the client in spite of the limitations, and patterns you can apply to make this work well for you.

Reviewer: ```suggestion
Inside a freshly initialized bare project, run `npx expo start`, and you can now run it in the client. Read this documentation to learn more about the limitations, why you might want to still use the client despite the limitations, and patterns you can apply to make this work well for you.
```

Unrelatd to PR but I think it's nice to improve this paragraph to imply what this page offers.
